lib/index.coffee

```diff
@@ -2,10 +2,51 @@ PomodoroAppView = require './pomodoro-app-view'
 {CompositeDisposable} = require 'atom'
 
 module.exports = PomodoroApp =
+  config:
+    startTime:
+      title: 'Start Time'
+      type: 'object'
+      properties:
+        minute:
+          type: 'integer'
+          default: 25
+          minimum: 0
+          maximum: 60
+        second:
```

Comment:
I was wondering, if we need seconds or not? We can default to 00 everytime. That will also simplify our user experience. What do you think?